src/test/java/com/lob/model/AddressTest.java

```diff
@@ -128,23 +128,4 @@ public void testAddressInvalidRequest() throws Exception {
         new Address.RequestBuilder().create();
     }
 
-    @Test(expected = RateLimitException.class)
-    public void testAddressRateLimit() throws Exception {
```

Comment:
Removing this test because it's an integration test verifying rate limiting with a specific account. Since we're no longer hard coding account keys, this test will never succeed. Therefore, I decided to remove the test entirely.